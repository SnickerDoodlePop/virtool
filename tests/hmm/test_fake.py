async def test_create_fake_hmm(fake2, spawn_client, snapshot):
    client = await spawn_client(authorize=True)

    hmm = await fake2.hmm.create(fake2.mongo)

    assert hmm == snapshot
