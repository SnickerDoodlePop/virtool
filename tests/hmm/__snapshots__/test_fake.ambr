# name: test_create_fake_hmm[uvloop]
  <class 'dict'> {
    '_id': 'RNvnAvOpyEVAoNGnVZQU',
    'cluster': 7851,
    'count': 3450,
    'entries': <class 'list'> [
      <class 'dict'> {
        'accession': 'qLUJyfwFVYySnPCaLuQI',
        'gi': 'azTmqTjDmYPxeqAWfCKC',
        'name': 'QCYFExFuDpjjFIyeNTWR',
        'organism': 'UWCuKoQSUEXExIZVPeFz',
      },
    ],
    'families': <class 'dict'> {
      'LdJCGRddpwdyFKunVyUD': 3332,
    },
    'genera': <class 'dict'> {
      'GRmMglPrQfhKcDVBJbHF': 9436,
    },
    'length': 1961,
    'mean_entropy': 49679341.1946931,
    'names': <class 'list'> [
      'YdJRrnFxjyvDAqsRnDaM',
    ],
    'total_entropy': 26035965871101.1,
  }
---
